src/definitions/abi/ret.rs

```diff
@@ -2,7 +2,8 @@ use super::*;
 
 pub const RET_FORWARDING_MODE_BYTE_IDX: usize = 28;
 
-const _: () = if RET_FORWARDING_MODE_BYTE_IDX != super::far_call::FAR_CALL_FORWARDING_MODE_BYTE_IDX {
```

Comment:
Is it possible to have this auto-formatted using CI?

Reply:
AFAIK @shahar4 already added the formatting CI to the main branch.

(but for historical branches - I'm aligning them as they are - as hopefully we'll not change these branches anymore)